.vscode/settings.json

```diff
@@ -81,8 +81,8 @@
   // Markdown
   "[markdown]": {
     "editor.defaultFormatter": "biomejs.biome",
-    "editor.unicodeHighlight.ambiguousCharacters": false,
-    "editor.unicodeHighlight.invisibleCharacters": false,
+    "editor.unicodeHighlight.ambiguousCharacters": true,
```

Comment:
Won't this only affect markdown files?

Reply:
I guess it will. It's `[markdown]` scope

Reply:
good point. i have the setting enabled globally so that might be why i didnt catch that